oak_restricted_kernel/src/mm/bitmap_frame_allocator.rs

```diff
@@ -185,18 +182,18 @@ mod tests {
     use x86_64::{structures::paging::Size4KiB, PhysAddr};
 
     fn create_allocator<const N: usize>(start: u64, end: u64) -> BitmapAllocator<Size4KiB, N> {
-        BitmapAllocator::<Size4KiB, N>::new(PhysFrame::range_inclusive(
+        BitmapAllocator::<Size4KiB, N>::new(PhysFrame::range(
             PhysFrame::from_start_address(PhysAddr::new(start)).unwrap(),
-            PhysFrame::from_start_address(PhysAddr::new(end)).unwrap(),
+            PhysFrame::from_start_address(PhysAddr::new(end)).unwrap() + 1,
         ))
     }
 
     fn create_frame(start: u64) -> PhysFrame<Size4KiB> {
         PhysFrame::from_start_address(PhysAddr::new(start)).unwrap()
     }
 
-    fn create_frame_range(start: u64, end: u64) -> PhysFrameRangeInclusive<Size4KiB> {
-        PhysFrame::range_inclusive(create_frame(start), create_frame(end))
+    fn create_frame_range(start: u64, end: u64) -> PhysFrameRange<Size4KiB> {
```

Comment:
Can this method also be modified to accept exclusive end? If not, could you rename it to make it more obvious, or at least add a comment?

Reply:
Yes, it can -- and I will do later. As I mentioned in the PR description, I wanted to do minimal changes to the tests in this PR.